product-manual/koala-agents.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "AI Agents"
+description: "Build a custom AI agent to automate your sales workflows"
+---
+
+## What is an agent?
+
+Agents are a way to automate your sales workflows. You can use them to simply answer questions about a profile or account, or to qualify leads and accounts.
+
+There are a few ways of creating an agent:
+- When setting up qualification in a play
+- AI agents on settings
+- Research agents
+
+Let's dive into each of these.
+
+## Qualification agents
```

Comment:
It's important to highlight (and repeat for both Qualification Agents and Research Agents) that an agent should be designed to answer 1 specific thing. Dont try to overload the agent with a bunch of questions. You can always combine multiple qualification agents to qualify leads/accounts, and this will produce better results than trying to ask too many things at once.

We can include some best practice about how to write your prompt, about when to use which model (and when in doubt use GPT 4o), etc.